pandas_sphinx_theme/layout.html

```diff
@@ -38,32 +38,42 @@
   <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
 {%- endblock %}
 {%- block content %}
+    {% block docs_navbar %}
     <nav class="navbar navbar-light navbar-expand-lg bg-light fixed-top bd-navbar" id="navbar-main">
       {%- include "docs-navbar.html" %}
     </nav>
+    {% endblock %}
 
     <div class="container-fluid">
       <div class="row flex-xl-nowrap">
+          {% block docs_sidebar %}
           <div class="col-12 col-md-3 col-xl-2 bd-sidebar">
               {%- include "docs-sidebar.html" %}
           </div>
+          {% endblock %}
 
+          {% block docs_toc %}
           <div class="d-none d-xl-block col-xl-2 bd-toc">
               {% if not (meta is not none and 'notoc' in meta) %}
                 {%- include "docs-toc.html" %}
               {% endif %}
             </div>
+          {% endblock %}
 
+          {% block docs_main %}
           <main class="col-12 col-md-9 col-xl-8 py-md-3 pl-md-5 bd-content" role="main">
+              {% block docs_body %}
```

Comment:
Do you plan to use this? (since you already have the "body" block as well)

Reply:
to be honest I am actually confused how this is working 😄 - the page content is inserted somewhere and I'd assume this would have been in the `{{body}}` block, but it seems like it is empty here. I didn't want to over-ride it because I worried that I'd be removing something I shouldn't haha...